mintapi/api.py

```diff
@@ -865,6 +865,19 @@ def __call_investments_endpoint(self):
             headers=self._get_api_key_header(),
         ).json()
 
+    def get_categories(self):
+        try:
+            categories = self.__call_categories_endpoint()["Category"]
+        except Exception:
```

Comment:
We should avoid hiding exceptions like this. This is not actually handling the error. It is hiding that it happened and it will raise a more ambiguous error later in add_parent_category_to_result. Either let the error happen or, if there is a way to proceed and give a meaningful result, do that.

Reply:
Roger that.  I'll reconsider this section.